src/purchases.ts

```diff
@@ -1473,6 +1473,13 @@ export default class Purchases {
     return Purchases.convertIntToRefundRequestStatus(refundRequestStatusInt);
   }
 
+  /**
+   * Presents the App Store sheet for managing subscriptions.
+   */
+  public static showManageSubscriptions(): Promise<void> {
```

Comment:
Would you mind adding some usages of the API in our `apitesters` project here: https://github.com/RevenueCat/react-native-purchases/blob/main/apitesters/purchases.ts

and in our purchase tester app with something like this: https://github.com/RevenueCat/react-native-purchases/blob/ee8e757cf0ebfebfc645ef80178f521468c5d8b4/examples/purchaseTesterTypescript/app/screens/HomeScreen.tsx#L235-L237 

If not that's fine, we can take care of that, just the last bits to get this mergable to `main` I think.